config/custom.js

```diff
@@ -19,5 +19,5 @@ module.exports.custom = {
   // mailgunSecret: 'key-testkeyb183848139913858e8abd9a3',
   // stripeSecret: 'sk_test_Zzd814nldl91104qor5911gjald',
   // …
-
+  baseURL: process.env.BASE_URL || 'http://localhost:1337',
```

Comment:
Requested change: We'll need something in here to populate this value for review environments like Heroku.

Reply:
Thinking maybe we don't need to do a bunch of work accommodate Heroku, considering that we'll be moving off of it soon enough? I assume that future environments that we'll be deploying to, we'll have more control over that baseURL, and setting it in the environment will work fine.

Reply:
What if we can used a helper/util function wherever we might use `baseURL`, instead of doing this, such that we have a more dynamic URL that works for review environments (for Heroku or Azure):

```
function getUrl(req, routeName) {
    const protocol = req.connection.encrypted ? 'https' : 'http';
    return (protocol + '://' + req.headers.host + sails.route(routeName));
```

Alternately, what if we made this line:

```
baseURL: process.env.BASE_URL || sails.getBaseUrl() || 'http://localhost:1337',
```

Realizing that this Sails function is deprecated, but hasn't been removed and it might meet our purposes for review environments right now.

Reply:
I like that ... didn't realize that getBaseUrl was still available. Saves a bunch of custom work. pushing a commit shortly.